otter/json_schema/group_schemas.py

```diff
@@ -321,6 +322,21 @@
 }
 
 
+def validate_launch_config_servicenet(lc):
+    """
+    Validate that if CLBs are provided, ServiceNet is also provided.
+    """
+    clb = any([lb.get('type', 'CloudLoadBalancer') == 'CloudLoadBalancer'
+               for lb in get_in(('args', 'loadBalancers'), lc, default=())])
+    servicenet = get_in(('args', 'server', 'networks'), lc, default=None)
```

Comment:
(Because if this is None, ServiceNet will definitely be available (by default), otherwise, ServiceNet has to be in this list, right?)